react/src/components/CustomizedImageList.tsx

```diff
@@ -45,6 +54,8 @@ const CustomizedImageList: React.FC<PropsWithChildren> = ({ children }) => {
   const [customizedImageListFetchKey, updateCustomizedImageListFetchKey] =
     useUpdatableState('initial-fetch');
   const [inFlightImageId, setInFlightImageId] = useState<string>();
+  const [imageSearch, setImageSearch] = useState('');
```

Comment:
It looks like you need imageSearch string based item filtering. 

[ref](https://github.com/lablup/backend.ai-webui/blob/188d31f8bd994635ee8e24dfc108663041a5d442/react/src/components/ImageList.tsx#L359-L394)